docs/basics/app-anatomy.md

```diff
@@ -228,6 +228,8 @@ Each module can expose gRPC endpoints, called [service methods](https://grpc.io/
 
 #### gRPC-gateway REST Endpoints
 
+The [module's REST interface](../building-modules/module-interfaces.md#legacy-rest) lets users generate transactions and query the state through REST calls to the application's [light client daemon](../core/node.md#lcd) (LCD). REST routes are defined in a file `client/rest/rest.go`, which is composed of:
```

Comment:
We shouldn't be using the nomenclature LCD -- that was abandoned many moons ago.

Reply:
It was never the light client daemon either (there was no light client) lol

Reply:
this was old doc, an error on my side when resolving conflicts, fixed!